guardrails/src/drivers/rest/routers/admin.py

```
@@ -0,0 +1,270 @@
+    loader = get_load_active_guardrail()
+
+    try:
+        materialized = await loader.execute()
+        await runtime.swap(materialized)
+
+        return ReloadResponse(
+            status="success",
+            revision=materialized.revision,
+            message=f"Reloaded configuration revision {materialized.revision}",
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)